app/views/layouts/application.html.erb

```diff
@@ -2,6 +2,8 @@
 <html>
   <head>
     <title>MaintenanceApp</title>
+    <link rel="stylesheet"
```

Comment:
I'll try later today, but I'm imagining I'd use the following line:

`<% stylesheet_link_tag "https://fonts.googleapis.com/css?family=Josefin+Sans|Roboto" %>`